### PR TITLE
Add import line to conan.tools examples

### PR DESCRIPTION
**Pull request by uilianries**
It's not clear how to import new `conan.tools`, let's try to keep it more explicit.

Related to https://github.com/conan-io/conan-center-index/issues/7377

### CODE REVIEW

**Comment by ericLemanissier** [comment]
This page has a typo too: https://docs.conan.io/en/latest/reference/conanfile/tools/files.html#conan-tools-rename it misses the `.files` in conan.tools.files.rename

File: reference/conanfile/tools/files.rst [hunk 1]
```diff
@@ -79,8 +95,10 @@ Utility functions to rename a file or folder *src* to *dst*. On Windows, it is v
 
 .. code-block:: python
 
+    from conan.tools.file import rename
+
     def source(self):
-      tools.rename(self, "lib-sources-abe2h9fe", "sources")  # renaming a folder
+      rename(self, "lib-sources-abe2h9fe", "sources")  # renaming a folder
```
Comment (memsharded) [inline reference/conanfile/tools/files.rst:101]:
```suggestion
        rename(self, "lib-sources-abe2h9fe", "sources")  # renaming a folder
```
Indent looks wrong

**Review by memsharded — APPROVED** [review]

**Comment by uilianries** [comment]
@ericLemanissier thank you! It's fixed now.

**Review by ericLemanissier — APPROVED** [review]

File: reference/conanfile/tools/files.rst [hunk 2]
```diff
@@ -68,7 +84,7 @@ Example of ``conandata.yml`` with different patches for different versions:
       "1.12.0":
         - patch_file: "patches/0001-buildflatbuffers-cmake.patch"
 
-conan.tools.rename()
+conan.tools.files.rename()
 --------------------
```
Comment (ericLemanissier) [inline reference/conanfile/tools/files.rst:88]:
```suggestion
--------------------------
```

File: reference/conanfile/tools/files.rst [hunk 3]
```diff
@@ -79,8 +95,10 @@ Utility functions to rename a file or folder *src* to *dst*. On Windows, it is v
 
 .. code-block:: python
 
+    from conan.tools.file import rename
```
Comment (danimtb) [inline reference/conanfile/tools/files.rst:98]:
```suggestion
    from conan.tools.files import rename
```

**Review by danimtb — APPROVED** [review]

**Pull request merged**